src/controllers/userController.ts

```diff
@@ -139,6 +139,10 @@ async function retrieveUserByEmail(req: Request, res: Response, next: NextFuncti
       attributes: {
         exclude: ['password', 'role'],
       },
+      include: {
+        association: User.associations.UserAnswers,
+        attributes: { exclude: ['id', 'content', 'userId', 'createdAt', 'updatedAt'] },
```

Comment:
I think can convert to this if you just want to include the `questionId` and nothing else. 
`attributes: { include: ['questionId'] },`